.github/workflows/dismiss-approvals.yml

```
@@ -0,0 +1,35 @@
+name: Dismiss PR approvals on non-maintainer pushes
+
+on:
+  pull_request:
+    types:
+      - synchronize
+
+jobs:
+  dismiss-approvals:
+    if: github.repository == 'Homebrew/homebrew-core'
+    runs-on: ubuntu-latest
+    steps:
+      - id: list-maintainers
+        name: Get maintainer list
+        env:
+          GITHUB_TOKEN: ${{secrets.HOMEBREW_GITHUB_PUBLIC_REPO_TOKEN}}
+        run: |
+          maintainers="$(
+            gh api \
+              --header 'Accept: application/vnd.github+json' \
+              --header 'X-GitHub-Api-Version: 2022-11-28' \
+              /orgs/Homebrew/teams/core/members |
+              jq '[.[] | .login]'
```

```suggestion
              /orgs/Homebrew/teams/core/members \
              --jq '[.[] | .login]'
```
If you only need one `jq` query then `gh` can handle it.